selfdrive/car/hyundai/interface.py

```diff
@@ -14,6 +14,18 @@
 ENABLE_BUTTONS = (Buttons.RES_ACCEL, Buttons.SET_DECEL, Buttons.CANCEL)
 
 
+def get_button_type(but):
+  if but == Buttons.RES_ACCEL:
```

Comment:
dict?

Reply:
Did that in this PR: https://github.com/commaai/openpilot/pull/24678 so this PR can be a two line change